veniq/baselines/semi/benefits.py

```diff
@@ -0,0 +1,103 @@
+from typing import Tuple, Dict, List
+from aibolit.ast_framework import AST, ASTNodeType, ASTNode
+from aibolit.utils.ast_builder import build_ast
+from aibolit.extract_method_baseline.extract_semantic import extract_method_statements_semantic, StatementSemantic
+from collections import Counter
```

Comment:
should come before aibolit imports

Reply:
Why?

Reply:
it's a convention https://www.python.org/dev/peps/pep-0008/#imports